docs/markdown/Introduction/media.md

```diff
@@ -4,10 +4,94 @@ slug: "media"
 excerpt: "Learn more about Pants and related topics from these talks, posts and podcasts featuring Pants contributors and users."
 hidden: false
 createdAt: "2021-04-18T17:27:18.361Z"
-updatedAt: "2022-04-05T22:02:38.457Z"
+updatedAt: "2022-09-08T22:02:38.457Z"
 ---
-Talks
------
+## Posts & Articles
+
+### Coinbase Blog
+
```

Comment:
None of the newly added content is italicized, whereas most of the existing content is italicized. See the preview: https://github.com/pantsbuild/pants/blob/ca460081ef0d99996719764f936853af906cbed0/docs/markdown/Introduction/media.md

Reply:
Fixed: the italicized quote for Astranis wasn't closed properly